lua/nvim-treesitter-refactor/highlight_current_scope.lua

```diff
@@ -16,10 +17,21 @@ function M.highlight_current_scope(bufnr)
   local current_scope = locals.containing_scope(node_at_point, bufnr)
 
   if current_scope then
-    local start_line = current_scope:start()
+    local config = configs.get_module "refactor.highlight_current_scope"
+    -- Highlight range [start_line, end_line) 0-based
+    local highlighter = function(start_line, end_line)
+      vim.api.nvim_buf_set_extmark(bufnr, current_scope_namespace, math.max(vim.fn.line "w0" - 1, start_line), 0, {
+        end_row = math.min(vim.fn.line "w$", end_line),
```

Comment:
please don't use functions that assume the current buffer for a function that explicitly takes a `bufnr` as an argument